qbraid_qir/squin/example.ipynb

```diff
@@ -0,0 +1,378 @@
+      "\u001b[90m  │ \u001b[0m%15 = \u001b[34mpy.indexing\u001b[0m.getitem(%1, %14)\u001b[30m : \u001b[0m\u001b[30m!py.Qubit\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m%16 = \u001b[34mpy.ilist\u001b[0m.new(\u001b[33mvalues=\u001b[0m(%15)){\u001b[33melem_type\u001b[0m=!Any}\u001b[30m : \u001b[0m\u001b[30m!py.IList\u001b[0m\u001b[30m[\u001b[0m\u001b[30m!py.Qubit\u001b[0m\u001b[30m, \u001b[0m\u001b[30mLiteral(1,int)\u001b[0m\u001b[30m]\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m      \u001b[34msquin.gate\u001b[0m.cx(\u001b[33mcontrols=\u001b[0m%13, \u001b[33mtargets=\u001b[0m%16)\n",
+      "\u001b[90m  │ \u001b[0m%17 = \u001b[34mpy.constant\u001b[0m.constant 2\u001b[90m : \u001b[0m\u001b[90m!py.int\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m%18 = \u001b[34mpy.indexing\u001b[0m.getitem(%1, %17)\u001b[30m : \u001b[0m\u001b[30m!py.Qubit\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m%19 = \u001b[34mpy.ilist\u001b[0m.new(\u001b[33mvalues=\u001b[0m(%18)){\u001b[33melem_type\u001b[0m=!Any}\u001b[30m : \u001b[0m\u001b[30m!py.IList\u001b[0m\u001b[30m[\u001b[0m\u001b[30m!py.Qubit\u001b[0m\u001b[30m, \u001b[0m\u001b[30mLiteral(1,int)\u001b[0m\u001b[30m]\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m%20 = \u001b[34mpy.constant\u001b[0m.constant 3\u001b[90m : \u001b[0m\u001b[90m!py.int\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m%21 = \u001b[34mpy.indexing\u001b[0m.getitem(%1, %20)\u001b[30m : \u001b[0m\u001b[30m!py.Qubit\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m%22 = \u001b[34mpy.ilist\u001b[0m.new(\u001b[33mvalues=\u001b[0m(%21)){\u001b[33melem_type\u001b[0m=!Any}\u001b[30m : \u001b[0m\u001b[30m!py.IList\u001b[0m\u001b[30m[\u001b[0m\u001b[30m!py.Qubit\u001b[0m\u001b[30m, \u001b[0m\u001b[30mLiteral(1,int)\u001b[0m\u001b[30m]\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m      \u001b[34msquin.gate\u001b[0m.cx(\u001b[33mcontrols=\u001b[0m%19, \u001b[33mtargets=\u001b[0m%22)\n",
+      "\u001b[90m  │ \u001b[0m%23 = \u001b[34mfunc\u001b[0m.const.none()\u001b[30m : \u001b[0m\u001b[30m!py.NoneType\u001b[0m\n",
+      "\u001b[90m  │ \u001b[0m      \u001b[34mfunc\u001b[0m\u001b[31m.\u001b[0m\u001b[31mreturn\u001b[0m %23\n",
+      "}\u001b[90m // func.func main\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Convert Cirq circuit to squin kernel using bloqade\n",
+    "main_loaded = lowering_module.load_circuit(circuit, kernel_name=\"main\")\n",
+    "\n",
+    "print(\"\\nSquin Kernel (from bloqade/cirq):\")\n",
+    "print(\"=\" * 60)\n",
+    "main_loaded.print()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1cbc7ec3",
+   "metadata": {},
+   "source": [
+    "## Run on Simulator and Compare output vectors"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "0b8735b3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "cirq-squin-output:\n",
```

Comment:
Shouldn't these output state vectors match exactly? I see a mis-match in the final state vector coefficients